sql/2024/fonts/performance/fonts_size.sql

```diff
@@ -0,0 +1,33 @@
+-- Section: Performance
+-- Question: What is the distribution of the file size?
+
+WITH
+fonts AS (
+  SELECT
+    client,
+    url,
+    AVG(PARSE_NUMERIC(JSON_EXTRACT_SCALAR(summary, '$.respBodySize'))) AS size
```

Comment:
We should probably take format into account here as just using WOFF vs. WOFF2 would throw off comparisons.

Reply:
(and in the other file size related queries as well)